packages/fluentui/react-northstar-prototypes/src/prototypes/chatMessageActions/index.tsx

```diff
@@ -0,0 +1,368 @@
+import {
+  Chat,
+  Provider,
+  Avatar,
+  ChatMessageProps,
+  ShorthandCollection,
+  ReactionProps,
+  Button,
+} from '@fluentui/react-northstar';
+
+import * as React from 'react';
+import ReactionPopup from './ReactionPopup';
+import { Ref } from '@fluentui/react-component-ref';
+import { AcceptIcon, EmojiIcon, LikeIcon } from '@fluentui/react-icons-northstar';
+import { keyboardKey, chatBehavior, menuAsToolbarBehavior, chatMessageBehavior } from '@fluentui/accessibility';
+import { PortalInner } from '@fluentui/react-northstar/src/components/Portal/PortalInner';
+
+const chatBehaviorOverride: any = () => {
+  const behavior = chatBehavior();
+
+  // doesn't work when chat message get only focus event
+  behavior.focusZone.props.shouldEnterInnerZone = event => false;
```

Comment:
need to override entering into the InnerZone
If it is not overridden, then "Enter" key will move focus inside the message...